src/scripts/main.js

```diff
@@ -1,7 +1,13 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  let element;
+
+  setTimeout(() => {
+    element = document.createElement('div');
+  }, 2 * 1000);
+
+  clearTimeout(element);
```

Comment:
The `clearTimeout` function is incorrectly used here. It should be used with a timer ID, not an element. You need to store the result of `setTimeout` in a variable and pass that variable to `clearTimeout` if you intend to cancel the timeout.